submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/MultipartFileSubmissionApiTest.kt

```diff
@@ -266,53 +268,90 @@ internal class MultipartFileSubmissionApiTest(
                 .withMessageContaining("Unsupported page tab format submission.txt")
         }
 
-        // TODO: pending fire files
-        private fun fileListFiles(fileListName: String, subFolder: String): List<ExtFile> =
-            if (mysqlMode) listOf() else listOf(
-                NfsFile("$fileListName.json", File(subFolder).resolve("Files/$fileListName.json")),
-                NfsFile("$fileListName.xml", File(subFolder).resolve("Files/$fileListName.xml")),
-                NfsFile("$fileListName.pagetab.tsv", File(subFolder).resolve("Files/$fileListName.pagetab.tsv"))
-            )
-
-        private fun submissionFiles(accNo: String, subFolder: String): List<ExtFile> =
-            if (mysqlMode) listOf() else listOf(
-                NfsFile("$accNo.json", File(subFolder).resolve("$accNo.json")),
-                NfsFile("$accNo.xml", File(subFolder).resolve("$accNo.xml")),
-                NfsFile("$accNo.pagetab.tsv", File(subFolder).resolve("$accNo.pagetab.tsv"))
-            )
-
         private fun assertSubmissionFiles(accNo: String, testFile: String) {
             val createdSub = submissionRepository.getExtByAccNo(accNo)
             val subFolder = "$submissionPath/${createdSub.relPath}"
 
-            val fileListName = createdSub.section.fileList?.fileName
-            val expectedTabFiles = if (mongoMode) {
-                if (enableFire) listOf() else listOf(
-                    NfsFile("$fileListName.json", File(subFolder).resolve("Files/$fileListName.json")),
-                    NfsFile("$fileListName.xml", File(subFolder).resolve("Files/$fileListName.xml")),
-                    NfsFile("$fileListName.pagetab.tsv", File(subFolder).resolve("Files/$fileListName.pagetab.tsv"))
-                )
-            } else listOf()
-
-            val submissionPageTabFiles = if (mongoMode) {
-                if (enableFire) listOf() else listOf(
-                    NfsFile("$accNo.json", File(subFolder).resolve("$accNo.json")),
-                    NfsFile("$accNo.xml", File(subFolder).resolve("$accNo.xml")),
-                    NfsFile("$accNo.pagetab.tsv", File(subFolder).resolve("$accNo.pagetab.tsv"))
-                )
-            } else listOf()
-
-            assertThat(createdSub.section.fileList).isEqualTo(ExtFileList("FileList", pageTabFiles = expectedTabFiles))
-            assertThat((createdSub.pageTabFiles)).isEqualTo(submissionPageTabFiles)
+            if (mongoMode)
```

Comment:
Same previous comment. I'm even wondering now if we should have different test classes that are enabled/disabled based on the properties 🤔. But this would be a "nice to have" thing that we could tackle in a separate ticket.